Tests/TestingTests/AttachmentTests.swift

```diff
@@ -34,7 +34,7 @@ import UIKit
 import UniformTypeIdentifiers
 #endif
 
-@Suite("Attachment Tests")
+@Suite("Attachment Tests", .disabled())
```

Comment:
I recall this test suite was disabled because of some bug you experienced on your machine. Since these tests are still (apparently) working on other systems besides yours, let's be sure to _not_ include this and keep the tests enabled elsewhere. You can continue to have `.disabled()` in your local working copy though of course